aggregators/config.go

```diff
@@ -140,6 +141,27 @@ func WithHarvestDelay(delay time.Duration) Option {
 	}
 }
 
+// WithMaxLookback configures the maximum duration that the
```

Comment:
```suggestion
// WithLookback configures the duration that the
```

It's always looking back this duration, so not sure having "maximum" in the name is helpful